### ISSUE TITLE
About AccessPoint

### ISSUE THREAD

**Issue by jianyoung** [isuPatches/WiseFy#104]
very impressive code but can WiseFy create a AccessPoint？

**Comment by jianyoung** [comment]
thanks but I mean open an AccessPoint for other devices，like ：
 Method method = wifiManager.getClass().getMethod("setWifiApEnabled",WifiConfiguration.class,Boolean.TYPE);
 method.invoke(wifiManager, config, true);

**Comment by isuPatches** [comment]
No, WiseFy does not currently use the 'setWifiApEnabled' API from WifiManager yet.  Can you describe more of your use case so it can be spec'ed out?
